src/test/java/uk/ac/ebi/eva/contigalias/ContigAliasApplicationTests.java

```diff
@@ -18,7 +18,9 @@
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
 
+@ActiveProfiles("test")
```

Comment:
It's ok as it is, but have you tried without this annotation? in our other projects we just specify the test application.properties and it just works.